pkg/upgrade/v1_22_0.go

```diff
@@ -19,10 +19,27 @@ func upgrade1_22_0(ctx context.Context, client client.Client, jaeger v1.Jaeger)
 		to:   "agent.tags",
 	}}
 
+	flagMapQuery := []deprecationFlagMap{
+		{
+			from: "downsampling.hashsalt",
+			to:   "",
+		},
+		{
+			from: "downsampling.ratio",
+			to:   "",
+		},
+	}
+
+	flagsAll := []deprecationFlagMap{{
+		from: "cassandra.tls.verify-host",
```

Comment:
This probably needs further handling to invert the flag. The names suggest that `verify-host: true` will verify the hostname, while `skip-host-verify: true` suggest that it will not verify the hostname... Ideally, the flag would be removed altogether if the new value is the same as the default value for the new flag: `verify-host: true` will be changed to `skip-host-verify: false`, which is the default value, so, the flag can be removed.